### PR TITLE
Prepare RH-TREX for RHTAP Onboarding

### PR DESCRIPTION
**Pull request by gurnben**
## Summary

This PR seeks to patch up internal references and add a new Containerfile targeted at RHTAP onboarding - this Containerfile is RH specific and RHTAP specific!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,4 +18,4 @@ LABEL name="ocm-example-service" \
       vendor="Red Hat" \
       version="0.0.1" \
       summary="OCM Example Service API" \
-      description="OCM Example Service API"
+      description="OCM Example Service API"
```
Comment (markturansky) [inline Dockerfile:21]: not OCM.

TRex is TAP Rest Example

File: Containerfile.rhtap [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    && \
+    microdnf clean all
+
+COPY --from=builder \
+    ocm-example-service \
+    /usr/local/bin/
+
+EXPOSE 8000
+
+ENTRYPOINT ["/usr/local/bin/ocm-example-service", "serve"]
```
Comment (markturansky) [inline Containerfile.rhtap:23]: binary name will change

File: Dockerfile (hunk 1 above)
Comment (gurnben) [reply Dockerfile:21]: Planning to resolve as part of https://issues.redhat.com/browse/HYPBLD-162!

File: Containerfile.rhtap (hunk 2 above)
Comment (gurnben) [reply Containerfile.rhtap:26]: Planning to resolve as part of https://issues.redhat.com/browse/HYPBLD-162!

Probably need a shared var that we can reference here!

**Pull request merged**